(doc) Added schema diagram from sql database

grisanu opened the pull request.

grisanu committed:
(feat) Added schema diagram from sql database

erinkav commented:
- Users table needs to store access token
- Images table should store an associated arc. We should not need the join table since each photo has a single arc it belongs to
- Based on what i've seen, fbook sends back many images and we'll select whichever size we want. Maybe we should store the size as well in the images table for later reference?

lessenadam commented:
Looks good. It hit me while looking at this that photos could show up in different stories, so the join table makes sense in my mind. Maybe just add a "criteria" field in addition to date_added on the arcs though?

grisanu committed:
(fix) Changed db schema in response to comments

-Removed join table
-Users table store access tokens
-Arcs table stores a search criteria and also the date range of the query to the fb api
-Images table stores images of different sizes
